src/screens/NameValueList.js

```diff
@@ -34,7 +34,7 @@ const NameValueListPage = () => (
         },
       ]}
       description="A name-value list displays a group of name-value pairs"
-      intrinsicElement="div"
+      intrinsicElement="dd"
```

Comment:
I think that this change may impact the instinct element type on the List prop type interface.

Reply:
How?

This isn't actually a change to the library, the docs are just wrong at present (as far as I can see). Notice that a `NameValueList` is a `Grid` rendered as a `dl`: https://github.com/grommet/grommet/blob/master/src/js/components/NameValueList/NameValueList.js#L56

Reply:
Now that I write that out I realize that this should have been `dl` not `dd`. Updated the PR.

Reply:
I'm sorry, when you say the "List prop type interface" I guess I'm not sure what you mean. There is `components/NameValueList/propTypes.js`, which perhaps should also be expanded to include `GridPropTypes` in the other PR. Is that what you meant?

Reply:
I agree with this being updated to `dl`, having the intrinsic element as `div` is incorrect for NameValueList